client/src/document.js

```diff
@@ -99,12 +99,7 @@ function RenderSideBar({ document }) {
 }
 
 function SidebarLeaf({ depth, title, content }) {
```

Comment:
You said...

> I couldn't find an example on the prod site that nested further than this.

Well, Stumptown isn't MDN :) Not yet at least :)
The way the `related_content` works in `stumptown-content` kinda assumes that the related content might be infinitely recursive. Perhaps we can just tackle that the day it actually becomes a thing. 

I think the only thing left to do is to delete `depth` from the destructured arguments.